testing/fake-dom.js

```diff
@@ -349,6 +349,15 @@ class FakeLocation {
   reload(forceReload) {
     this.change_({reload: true, forceReload});
   }
+
+  /**
+   * Resets the URL without firing any events or triggering a history
+   * entry.
+   * @param {string} href
+   */
+  setForTesting(href) {
```

Comment:
Everything here is for testing :) Let's call it something like `setQuiet`. Generally, the intention here is if you need an initially set location, you declare it in `describe`. That might not be applicable in all cases, so I'm fine with the change.